src/defaults/fileIcons.ts

```diff
@@ -38,6 +38,20 @@ export const fileIcons: FileIcons = {
       'apollo.config.ts',
     ],
   },
+  'astro': {
+    fileExtensions: ['astro'],
+    languageIds: ['astro'],
+  },
+  'astro-config': {
+    fileNames: [
+      'astro.config.js',
+      'astro.config.mjs',
+      'astro.config.cjs',
+      'astro.config.ts',
+      'astro.config.cts',
+      'astro.config.mts',
```

Comment:
These extensions don't look right? I didn't say `mts` or `cts`... those don't appear [to be supported by Astro](https://docs.astro.build/en/guides/configuring-astro/#supported-config-file-types).

Reply:
`mts` and `cts` are just ways to compile ts explicitely to cjs/mjs, i put them as a future-proof thing that involve 0 risk

Reply:
👍